utils/filepattern-generator-plugin/bumpversion.cfg

```diff
@@ -0,0 +1,10 @@
+[bumpversion]
+current_version = 0.1.6
```

Comment:
This is out of sync with the VERSION file.

Reply:
Done.